clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -1616,6 +1616,7 @@ check_schedule_file()
     sched.emplace_back(TID_BASE, TIMESTAMP_BASE + 1, CPU_BASE + 1, 2);
     sched.emplace_back(TID_BASE + 1, TIMESTAMP_BASE + 2, CPU_BASE, 1);
     sched.emplace_back(TID_BASE + 2, TIMESTAMP_BASE + 3, CPU_BASE + 2, 3);
+    sched.emplace_back(TID_BASE + 2, TIMESTAMP_BASE + 3, CPU_BASE + 2, 4);
```

Comment:
Add to the PR description that a test is added which fails without the fix.

Reply:
Done.